upload_to_config_bucket.tf

```diff
@@ -71,13 +76,8 @@ resource "aws_s3_bucket_object" "installer_sh" {
 data "template_file" "installer_sh" {
   template = file(format("%s/installer.sh", path.module))
   vars = {
-    VERSION                 = local.adg_version[local.environment]
-    ADG_LOG_LEVEL           = local.adg_log_level[local.environment]
-    ENVIRONMENT_NAME        = local.environment
-    S3_COMMON_LOGGING_SHELL = format("s3://%s/%s}", data.terraform_remote_state.common.outputs.config_bucket.id, data.terraform_remote_state.common.outputs.application_logging_common_file.s3_id)
-    S3_LOGGING_SHELL        = format("s3://%s/%s", data.terraform_remote_state.common.outputs.config_bucket.id, aws_s3_bucket_object.logging_script.key)
-    full_proxy              = data.terraform_remote_state.internet_egress.outputs.internet_proxy.http_address
-    full_no_proxy           = "127.0.0.1,localhost,169.254.169.254,*.s3.eu-west-2.amazonaws.com,s3.eu-west-2.amazonaws.com,sns.eu-west-2.amazonaws.com,sqs.eu-west-2.amazonaws.com,eu-west-2.queue.amazonaws.com,glue.eu-west-2.amazonaws.com,sts.eu-west-2.amazonaws.com,*.eu-west-2.compute.internal,dynamodb.eu-west-2.amazonaws.com"
+    full_proxy    = data.terraform_remote_state.internet_egress.outputs.internet_proxy.http_address
```

Comment:
is this both http & https?  It's confusing from what's here, and its naming etc.